demisto_sdk/__main__.py

```diff
@@ -3076,6 +3083,116 @@ def update_content_graph(
         )
 
 
+@main.command()
+@click.help_option("-h", "--help")
+@click.option(
+    "-i",
+    "--input",
+    help="The path to the input file to run the command on.",
+    multiple=True,
+    type=click.Path(path_type=Path),
+)
+@click.option(
+    "-g",
+    "--use-git",
+    help="Whether to use git to determine which files to run the command on",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "-s",
+    "--staged",
+    help="Whether to run on staged files only",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "-a",
+    "--all-files",
+    help="Whether to run precommit on all files",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "--no-fix",
+    help="Whether to skip fixing code files",
+    is_flag=True,
+)
+@click.option(
+    "-t",
+    "--test",
+    help="Whether to run tests",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "--skip",
+    help="A comma separated list of hooks of precommit hooks to skip",
+)
+@click.option(
+    "--force-run-hooks",
```

Comment:
what is this used for?

Reply:
To run hooks that are disabled by deafult, such as `validate` and `format`.
Maybe a better solution is to add an option for each

Reply:
`force` doesn't sound suitable, an option for each sounds good